.expeditor/scripts/shared.ps1

```diff
@@ -69,22 +84,35 @@ function Install-RustToolchain($Toolchain) {
   }
 }
 
+function Test-Port([string]$ComputerName, [int]$Port) {
+  If (Get-Command Test-NetConnection -ErrorAction SilentlyContinue) {
+    (Test-NetConnection -ComputerName $ComputerName -Port $Port).TcpTestSucceeded
+  } else {
+    Test-Connection -ComputerName $ComputerName -TCPPort $Port
+  }
+}
+
 function Wait-Supervisor($Timeout=1) {
   Write-Host "Waiting up to $Timeout seconds for Supervisor to start..."
   $startTime = [DateTime]::Now
-  $success = (Test-NetConnection -ComputerName localhost -Port 9631).TcpTestSucceeded
+  $success = Test-Port -ComputerName localhost -Port 9631
   while(!$success) {
       Start-Sleep -Seconds 1
       $timeTaken = [DateTime]::Now.Subtract($startTime)
       if($timeTaken.TotalSeconds -ge $Timeout) {
           Write-Error "Timed out waiting $Timeout seconds for Supervisor to start"
           break
       }
-      $success = (Test-NetConnection -ComputerName localhost -Port 9631).TcpTestSucceeded
+      $success = Test-Port -ComputerName localhost -Port 9631
   }
   if($success) { Write-Host "Supervisor is now running." }
 }
 
+function Start-Supervisor($Timeout=1) {
+  hab sup run --no-color &
```

Comment:
The tailing `&` really makes the windows tests unhappy. You might try something like the pattern in https://github.com/habitat-sh/habitat/blob/master/test/end-to-end/test_supervisor_load_service.ps1:

Start using:
```
$job = start-job { hab sup run }
```

Then:
```
    AfterAll {
        hab svc unload core/nginx
        Stop-Job -Job $job
        Remove-Job -Job $job
    }
```

Reply:
Yeah, if we want our e2e tests to support "Windows Powershell" there will be a few places of cleanup, but I am hoping we can make the e2e tests solely target powershell core. See this [comment](https://github.com/habitat-sh/habitat/pull/7148#discussion_r342723106).